manage-data/migrate/migrate-with-logstash.md

```diff
@@ -0,0 +1,95 @@
+---
+navigation_title: Migrate Elastic Cloud Hosted data to Serverless with Logstash
+applies_to:
+  stack: ga
+  deployment:
+    ess: ga
+    ece: ga
+products:
+  - id: elasticsearch
+  - id: logstash
+  - id: cloud-hosted
+  - id: cloud-enterprise
+  - id: cloud-kubernetes
+---
+
+# Migrate {{ech}} data to {{serverless-full}} with {{ls}} [migrate-with-ls]
+
+You can use {{ls}} to migrate data from an {{ech}} deployment to an {{serverless-full}} project. 
+Familiarity with {{ech}}, {{es}}, and {{ls}} is helpful, but not required. 
+
+:::{admonition} Basic migration
+This guide focuses on a basic data migration scenario for moving static data from an {{ech}} deployment to a {{serverless-full}} project. Dashboards, visualizations, pipelines, templates, and other {{kib}} assets must be migrated separately using the {{kib}} [export/import APIs](https://www.elastic.co/docs/api/doc/kibana/group/endpoint-saved-objects) or recreated manually.
+:::
+
+## Prerequisites [migrate-prereqs]
+
+- {{ech}} deployment with data to migrate
+- [{{serverless-full}}](/deploy-manage/deploy/elastic-cloud/serverless.md) project configured and running
+- {{ls}} [installed](https://www.elastic.co/downloads/logstash) on your local machine or server 
+- API keys in {{ls}} format for authentication with both deployments
+
+
+## Process overview [migration-overview]
+* [Configure {{ls}}](#configure-ls)
+* [Run {{ls}}](#run-ls)
+* [Verify data migration](#verify-migration)
+
+
+## Step 1: Configure {{ls}} [configure-ls]
```

Comment:
Should the first step be a brief blurb about Logstash? Incl. a link to installation or quick start docs?